### PR TITLE
mkfs*: add Chinese translation

### PR DESCRIPTION
**Pull request by kfatyuip**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/linux/mkfs.btrfs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# mkfs.btrfs
+
+> 创建一个BTRFS文件系统。
+> 默认情况下是 `raid1`，指定了数据块的两份拷贝分布在两个不同的设备上。
+> 更多信息： <https://btrfs.readthedocs.io/en/latest/mkfs.btrfs.html>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.btrfs.md:5]: A general copywriting rule in Chinese is adding a space between full-width and half-width characters.
```suggestion
> 创建一个 BTRFS 文件系统。
> 默认情况下是 `raid1`，指定了数据块的两份拷贝分布在两个不同的设备上。
> 更多信息：<https://btrfs.readthedocs.io/en/latest/mkfs.btrfs.html>.
```

File: pages.zh/linux/mkfs.btrfs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- 在单个设备上创建一个btrfs文件系统：
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.btrfs.md:7]:
```suggestion
- 在单个设备上创建一个 btrfs 文件系统：
```

File: pages.zh/linux/mkfs.btrfs.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+`sudo mkfs.btrfs --metadata single --data single {{/dev/sda}}`
+
+- 在多个设备上使用raid1创建一个btrfs文件系统：
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.btrfs.md:11]:
```suggestion
- 在多个设备上使用 raid1 创建一个 btrfs 文件系统：
```

File: pages.zh/linux/mkfs.cramfs.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# mkfs.cramfs
+
+> 创建一个ROM文件系统，放置在分区内。
+> 更多信息： <https://manned.org/mkfs.cramfs>.
+
+- 在设备b的分区1内创建一个ROM文件系统 (`sdb1`):
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.cramfs.md:6]:
```suggestion
- 在设备b的分区1内创建一个 ROM 文件系统（`sdb1`）：
```

File: pages.zh/linux/mkfs.cramfs.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+`mkfs.cramfs {{/dev/sdb1}}`
+
+- 创建一个带有卷名的ROM文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.cramfs.md:10]:
```suggestion
- 创建一个带有卷名的 ROM 文件系统：
```

File: pages.zh/linux/mkfs.cramfs.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# mkfs.cramfs
+
+> 创建一个ROM文件系统，放置在分区内。
+> 更多信息： <https://manned.org/mkfs.cramfs>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.cramfs.md:4]:
```suggestion
> 创建一个 ROM 文件系统，放置在分区内。
> 更多信息：<https://manned.org/mkfs.cramfs>.
```

File: pages.zh/linux/mkfs.exfat.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# mkfs.exfat
+
+> 在分区内创建一个exFAT文件系统。
+> 更多信息： <https://manned.org/mkfs.exfat>.
+
+- 在设备b的分区1内创建一个exFAT文件系统 (`sdb1`):
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.exfat.md:6]:
```suggestion
- 在设备b的分区1内创建一个 exFAT 文件系统（`sdb1`）：
```

File: pages.zh/linux/mkfs.exfat.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# mkfs.exfat
+
+> 在分区内创建一个exFAT文件系统。
+> 更多信息： <https://manned.org/mkfs.exfat>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.exfat.md:4]:
```suggestion
> 在分区内创建一个 exFAT 文件系统。
> 更多信息：<https://manned.org/mkfs.exfat>.
```

File: pages.zh/linux/mkfs.exfat.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+- 在设备b的分区1内创建一个exFAT文件系统 (`sdb1`):
+
+`mkfs.exfat {{/dev/sdb1}}`
+
+- 创建一个带有卷名的文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.exfat.md:10]:
```suggestion
- 创建一个带有卷名的文件系统：
```

File: pages.zh/linux/mkfs.exfat.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+`mkfs.exfat -n {{volume_name}} {{/dev/sdb1}}`
+
+- 创建一个带有卷ID的文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.exfat.md:14]:
```suggestion
- 创建一个带有卷 ID 的文件系统：
```

File: pages.zh/linux/mkfs.ext4.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
+# mkfs.ext4
+
+> 在分区内创建一个ext4文件系统。
+> 更多信息： <https://manned.org/mkfs.ext4>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.ext4.md:4]:
```suggestion
> 在分区内创建一个 ext4 文件系统。
> 更多信息：<https://manned.org/mkfs.ext4>.
```

File: pages.zh/linux/mkfs.ext4.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+
+- 在设备b的分区1内创建一个ext4文件系统 (`sdb1`):
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.ext4.md:6]:
```suggestion
- 在设备b的分区1内创建一个 ext4 文件系统（`sdb1`）：
```

File: pages.zh/linux/mkfs.ext4.md [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo mkfs.ext4 {{/dev/sdb1}}`
+
+- 创建一个带有卷标签的ext4文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.ext4.md:10]:
```suggestion
- 创建一个带有卷标签的 ext4 文件系统：
```

File: pages.zh/linux/mkfs.f2fs.md [hunk 14]
```diff
@@ -0,0 +1,12 @@
+# mkfs.f2fs
+
+> 在分区内创建一个F2FS文件系统。
+> 更多信息： <https://manned.org/mkfs.f2fs>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.f2fs.md:4]:
```suggestion
> 在分区内创建一个 F2FS 文件系统。
> 更多信息：<https://manned.org/mkfs.f2fs>.
```

File: pages.zh/linux/mkfs.f2fs.md [hunk 15]
```diff
@@ -0,0 +1,12 @@
...
+
+- 在设备b的分区1内创建一个F2FS文件系统 (`sdb1`):
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.f2fs.md:6]:
```suggestion
- 在设备b的分区1内创建一个 F2FS 文件系统（`sdb1`）：
```

File: pages.zh/linux/mkfs.f2fs.md [hunk 16]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo mkfs.f2fs {{/dev/sdb1}}`
+
+- 创建一个带有卷标签的F2FS文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.f2fs.md:10]:
```suggestion
- 创建一个带有卷标签的 F2FS 文件系统：
```

File: pages.zh/linux/mkfs.fat.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
+# mkfs.fat
+
+> 在分区内创建一个MS-DOS文件系统。
+> 更多信息： <https://manned.org/mkfs.fat>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.fat.md:4]:
```suggestion
> 在分区内创建一个 MS-DOS 文件系统。
> 更多信息：<https://manned.org/mkfs.fat>.
```

File: pages.zh/linux/mkfs.fat.md [hunk 18]
```diff
@@ -0,0 +1,20 @@
...
+
+- 在设备b的分区1内创建一个FAT文件系统 (`sdb1`):
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.fat.md:6]:
```suggestion
- 在设备b的分区1内创建一个 FAT 文件系统（`sdb1`）：
```

File: pages.zh/linux/mkfs.fat.md [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+
+`mkfs.fat {{/dev/sdb1}}`
+
+- 创建一个带有卷名的文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.fat.md:10]:
```suggestion
- 创建一个带有卷名的文件系统：
```

File: pages.zh/linux/mkfs.fat.md [hunk 20]
```diff
@@ -0,0 +1,20 @@
...
+
+`mkfs.fat -n {{volume_name}} {{/dev/sdb1}}`
+
+- 创建一个带有卷ID的文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.fat.md:14]:
```suggestion
- 创建一个带有卷 ID 的文件系统：
```

File: pages.zh/linux/mkfs.fat.md [hunk 21]
```diff
@@ -0,0 +1,20 @@
...
+
+`mkfs.fat -i {{volume_id}} {{/dev/sdb1}}`
+
+- 使用5个而不是2个文件分配表:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.fat.md:18]:
```suggestion
- 使用 5 个而不是 2 个文件分配表：
```

File: pages.zh/linux/mkfs.md [hunk 22]
```diff
@@ -0,0 +1,17 @@
+# mkfs
+
+> 在硬盘分区上建立一个Linux文件系统。
+> 该命令已被废弃，建议使用特定文件系统的 mkfs.<type> 工具。
+> 更多信息： <https://manned.org/mkfs>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.md:5]:
```suggestion
> 在硬盘分区上建立一个 Linux 文件系统。
> 该命令已被废弃，建议使用特定文件系统的 mkfs.<type> 工具。
> 更多信息：<https://manned.org/mkfs>.
```

File: pages.zh/linux/mkfs.md [hunk 23]
```diff
@@ -0,0 +1,17 @@
...
+
+- 在分区上建立一个Linux ext2文件系统：
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.md:7]:
```suggestion
- 在分区上建立一个 Linux ext2 文件系统：
```

File: pages.zh/linux/mkfs.minix.md [hunk 24]
```diff
@@ -0,0 +1,8 @@
+# mkfs.minix
+
+> 在分区内创建一个Minix文件系统。
+> 更多信息： <https://manned.org/mkfs.minix>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.minix.md:4]:
```suggestion
> 在分区内创建一个 Minix 文件系统。
> 更多信息：<https://manned.org/mkfs.minix>.
```

File: pages.zh/linux/mkfs.minix.md [hunk 25]
```diff
@@ -0,0 +1,8 @@
...
+
+- 在设备b的分区1内创建一个Minix文件系统 (`sdb1`):
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.minix.md:6]:
```suggestion
- 在设备b的分区1内创建一个 Minix 文件系统（`sdb1`）：
```

File: pages.zh/linux/mkfs.ntfs.md [hunk 26]
```diff
@@ -0,0 +1,16 @@
+# mkfs.ntfs
+
+> 在分区内创建一个NTFS文件系统。
+> 更多信息： <https://manned.org/mkfs.ntfs>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.ntfs.md:4]:
```suggestion
> 在分区内创建一个 NTFS 文件系统。
> 更多信息：<https://manned.org/mkfs.ntfs>.
```

File: pages.zh/linux/mkfs.ntfs.md [hunk 27]
```diff
@@ -0,0 +1,16 @@
...
+
+- 在设备b的分区1内创建一个NTFS文件系统 (`sdb1`):
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.ntfs.md:6]:
```suggestion
- 在设备b的分区1内创建一个 NTFS 文件系统（`sdb1`）：
```

File: pages.zh/linux/mkfs.ntfs.md [hunk 28]
```diff
@@ -0,0 +1,16 @@
...
+
+`mkfs.ntfs {{/dev/sdb1}}`
+
+- 创建一个带有卷标签的文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.ntfs.md:10]:
```suggestion
- 创建一个带有卷标签的文件系统：
```

File: pages.zh/linux/mkfs.ntfs.md [hunk 29]
```diff
@@ -0,0 +1,16 @@
...
+
+`mkfs.ntfs -L {{volume_label}} {{/dev/sdb1}}`
+
+- 创建一个带有特定UUID的文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.ntfs.md:14]:
```suggestion
- 创建一个带有特定 UUID 的文件系统：
```

File: pages.zh/linux/mkfs.vfat.md [hunk 30]
```diff
@@ -0,0 +1,20 @@
+# mkfs.vfat
+
+> 在分区内创建一个MS-DOS文件系统。
+> 更多信息： <https://manned.org/mkfs.vfat>.
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.vfat.md:4]:
```suggestion
> 在分区内创建一个 MS-DOS 文件系统。
> 更多信息：<https://manned.org/mkfs.vfat>.
```

File: pages.zh/linux/mkfs.vfat.md [hunk 31]
```diff
@@ -0,0 +1,20 @@
...
+
+- 在设备b的分区1内创建一个vfat文件系统 (`sdb1`):
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.vfat.md:6]:
```suggestion
- 在设备b的分区1内创建一个 vfat 文件系统（`sdb1`）：
```

File: pages.zh/linux/mkfs.vfat.md [hunk 32]
```diff
@@ -0,0 +1,20 @@
...
+
+`mkfs.vfat {{/dev/sdb1}}`
+
+- 创建一个带有卷名的文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.vfat.md:10]:
```suggestion
- 创建一个带有卷名的文件系统：
```

File: pages.zh/linux/mkfs.vfat.md [hunk 33]
```diff
@@ -0,0 +1,20 @@
...
+
+`mkfs.vfat -n {{volume_name}} {{/dev/sdb1}}`
+
+- 创建一个带有卷ID的文件系统:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.vfat.md:14]:
```suggestion
- 创建一个带有卷 ID 的文件系统：
```

File: pages.zh/linux/mkfs.vfat.md [hunk 34]
```diff
@@ -0,0 +1,20 @@
...
+
+`mkfs.vfat -i {{volume_id}} {{/dev/sdb1}}`
+
+- 使用5个而不是2个文件分配表:
```
Comment (blueskyson) [inline pages.zh/linux/mkfs.vfat.md:18]:
```suggestion
- 使用 5 个而不是 2 个文件分配表：
```

File: pages.zh/linux/mkfs.btrfs.md (hunk 1 above)
Comment (kfatyuip) [reply pages.zh/linux/mkfs.btrfs.md:5]: Thanks for reviewing!

**Review by einverne — CHANGES_REQUESTED** [review]
hi @kfatyuip thank you for the new pages, I left a suggestion.

File: pages.zh/linux/mkfs.cramfs.md [hunk 35]
```diff
@@ -0,0 +1,12 @@
...
+
+- 在设备b的分区1内创建一个 ROM 文件系统（`sdb1`）：
```
[the remark this answers is not in the document]
Comment (kfatyuip) [reply pages.zh/linux/mkfs.cramfs.md]: thanks!

**Review by blueskyson — APPROVED** [review]

**Review by einverne — APPROVED** [review]
LGTM.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**